### PR TITLE
Xena: Add note on standard monitoring config usage

### PR DESCRIPTION
**Pull request by cityofships**
- Backport SMART monitoring notes from Yoga
- Add note on enabling standard monitoring config.

### CODE REVIEW

File: doc/source/configuration/monitoring.rst [hunk 1]
```diff
@@ -10,4 +14,52 @@ The configuration options can be found in
...
+
+.. code-block:: console
+
+    cd $KAYOBE_CONFIG_PATH
+    ln -s kolla/config/grafana/ environments/$KAYOBE_ENVIRONMENT/kolla/config/
+    ln -s kolla/config/prometheus/ environments/$KAYOBE_ENVIRONMENT/kolla/config/
+
+and commit them to the config repository.
+
+SMART Drive Monitoring
```
Comment (markgoddard) [inline doc/source/configuration/monitoring.rst:45]: We also don't have the code for this in xena... I'll propose a backport.

Comment (markgoddard) [reply doc/source/configuration/monitoring.rst:45]: Could you rebase onto https://github.com/stackhpc/stackhpc-kayobe-config/pull/301, or wait for it to merge?

File: doc/source/configuration/monitoring.rst [hunk 2]
```diff
@@ -1,4 +1,8 @@
-========================
+===========
+Monitoring
+===========
+
+
 Monitoring Configuration
 ========================
```
Comment (markgoddard) [inline doc/source/configuration/monitoring.rst:7]: Let's provide a bit more information about what's included in the stock config. Which services are enabled? What custom config is provided (probably don't need to list dashboards or rules files, but at least note that they are present).

File: doc/source/configuration/monitoring.rst [hunk 3]
```diff
@@ -10,4 +14,52 @@ The configuration options can be found in
...
+In order to enable stock monitoring configuration within a particular
+environment, create the following symbolic links:
+
+.. code-block:: console
+
+    cd $KAYOBE_CONFIG_PATH
+    ln -s kolla/config/grafana/ environments/$KAYOBE_ENVIRONMENT/kolla/config/
+    ln -s kolla/config/prometheus/ environments/$KAYOBE_ENVIRONMENT/kolla/config/
+
+and commit them to the config repository.
```
Comment (markgoddard) [inline doc/source/configuration/monitoring.rst:43]: There's also the rsyslog custom playbook. How and when should it be used?

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: doc/source/configuration/monitoring.rst (hunk 3 above)
Comment (cityofships) [reply doc/source/configuration/monitoring.rst:43]: Added

File: doc/source/configuration/monitoring.rst (hunk 2 above)
Comment (cityofships) [reply doc/source/configuration/monitoring.rst:7]: Added

**Review by markgoddard — APPROVED** [review]
Thanks

**Pull request merged**